src/icons/bell_icon.svg

```diff
@@ -4,7 +4,7 @@
 	 x="0px" y="0px" width="24px" height="24px" viewBox="0 0 24 24"
 	 style="enable-background:new 0 0 24 24;" xml:space="preserve">
 <style type="text/css">
-	.st0{fill:#595959;}
+	.st0{fill:context-fill}
```

Comment:
Is this option going to work if/when we go to AMO? I thought this was reserved for mozilla-signed addons?

Also, have a different approach over here in #388 which isn't necessarily mutually exclusive to this one

Reply:
@lmorchard here's the DXR for this. Do we have to change add-on IDs b/c that's how the permission is set

https://dxr.mozilla.org/mozilla-central/source/layout/svg/SVGContextPaint.cpp#69

Reply:
Okay, so it relies on the add-on ID and not the signing key. So, from there I guess my question is: Should we still be leaning on a special case that other WebExtensions don't have if/when this thing leaves Test Pilot?